.github/workflows/build.yml

```diff
@@ -22,6 +22,10 @@ jobs:
           check_name: Example JUnit Test Report
           report_paths: '**/surefire-reports/TEST-*.xml'
           summary: '<table><thead><tr><th> Application (src/applications) </th></tr></thead><tbody><tr><td> test </td></tr></tbody></table>'
+          # Since the template delimiters used internally clash with those
+          # used by GitHub Actions, escape them by wrapping them in a GHA
+          # string expression
+          check_title_template: ${{ '${{SUITE_NAME}} | ${{TEST_NAME}}' }}
```

Comment:
Given the template change, this can now also be updated and simplified

Reply:
Aha good spot, I missed that.

Reply:
Fixed in https://github.com/mikepenz/action-junit-report/pull/546/commits/888386a4202477addec901dce17fc417bea38f9b